certified-connectors/FlowForma V2/apiDefinition.swagger.json

```diff
@@ -0,0 +1,585 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FlowForma V2",
+    "description": "FlowForma Business Process Management (BPM) tools for Office 365, enable users to build forms & workflow, automating processes without code.",
+    "contact": {
+      "name": "FlowForma Limited",
+      "url": "https://www.flowforma.com",
+      "email": "support@flowforma.com"
+    },
+    "version": "v1"
```

Comment:
@FlowForma This should be `"V2"`.

Reply:
No, this shouldn't. the "V2" is a part of the connector's name, not the version. The API is completely new for this, so the swagger.json version should still be "v1".